lib/src/base-styles/_nx-page.scss

```diff
@@ -11,8 +11,10 @@
   display: grid;
   grid-template-columns: auto 1fr auto;
   grid-template-areas:
-    "title tags actions"
-    "description description description";
+    "title sub-title actions"
+    "description description description"
+    "tags tags tags";
+  column-gap: $nx-spacing-md;
```

Comment:
You've got this and also left margin on the nx-btn-bar, adding up to a total of 40px (should be 24)
![image](https://user-images.githubusercontent.com/3630091/115409198-8957f300-a1bf-11eb-948f-24b2455b2d56.png)

Reply:
> You've got this and also left margin on the nx-btn-bar, adding up to a total of 40px (should be 24)
> ![image](https://user-images.githubusercontent.com/3630091/115409264-94ab1e80-a1bf-11eb-8a87-8c52b8e72ece.png)

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/efd0b5bd7ecab9f0a1acd4d3bbdd6f895fffed5b